### PR TITLE
add newcomers page

### LINKED ISSUE #1064: [Layer5] Add a newcomers page to the site
**Current Behavior**

**Desired Behavior**

Take a look at all the things which need to be added.
- [x] Add the newcomers meeting detail available on [this page](https://docs.google.com/document/d/1aHarIyIL__HRq_4klYOUsEXP5SQLlmb50W17yKdNmpU/edit#heading=h.69qskjv558bk)
- [x] Add all the tutorials made during Hacktoberfest
- [x] Newcomers resources for getting started
- [x] Newcomers meeting table at the bottom of the page
- [ ] Newcomers Questionnaire

The working link should also be visible in the nav bar under the Community section.

Tagging @leecalcote @chandrashritii @Ruth-ikegah if I am missing something.

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Signed-off-by: Navendu Pottekkat navendupottekkat@gmail.com

**Description**

**This is a draft PR and is subject to change after review.**

This PR adds a _Newcomers_ page under the _Community_ dropdown.

The content is based on the [Newcomer](https://docs.google.com/document/d/1tpg2sLxirozNt3Ofr3GdM002f9rExp74EqrsGZBU710/edit#) and [Contributing](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit#heading=h.rcr9t5pnt3bw) Guides.

Added features:

- [x] Quick links to Layer5 pages and docs across the web added.
- [x] Instructions to newcomers added.
- [x] Hacktoberfest tutorials added.

This PR fixes #1064 

**Notes for Reviewers**

Please check this out and let me know of any changes to be made.

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by navendu-pottekkat** [comment]
@Ruth-ikegah @tripathyprateek Please check it out and let me know if I need to change anything or add any more content.

After that, I will change it from the draft and we can merge.

File: _includes/newcomers/newcomers.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <td><a href="" target="_blank">Calender</a> | <a href="" target="_blank">Slack</a> | <a href=""
+                target="_blank">GitHub</a> | <a href="" target="_blank">YouTube</a> | <a href=""
+                target="_blank">Drive</a></td>
+        <td><a href="" target="_blank">First-timers' Welcome Guide</a></td>
+        <td><a href="" target="_blank">help-wanted</a></td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td><a href="" target="_blank">Welcome Newcomers!</a></td>
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:28]: `<a href = "https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit#heading=h.504assxl7gpf"`

This is the link to the Repository overview

File: _includes/newcomers/newcomers.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    </tr>
+    <tr>
+        <td><a href="" target="_blank">Weekly Zoom on Thursdays at 8AM Central</a></td>
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:31]: <a href = "meet.layer5.io/newcomers">

Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:31]: `<a href = "meet.layer5.io/newcomers">`

File: _includes/newcomers/newcomers.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
+        <td><a href="" target="_blank">good first issue</a></td>
+    </tr>
+</table>
+
+<h3>How to contribute?</h3>
+
+<p>
+    The following set of steps outlines the process by which you can openly engage, learn, and participate in the broad
+    set of open source projects at Layer5. If at any time you get stuck, please seek help in the #first-timers channel
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:41]: There is no #first-timers channel, it should be #newcomers channel

File: _includes/newcomers/newcomers.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        the assignee section on the issue. If it hasn't been, signal your interest by commenting on the issue so that it
+        can be assigned to you.
+    </li>
+
+</ol>
+
+<h4>Working on your issue</h4>
+
+<p>Once an issue has been assigned to you, it’s time to get started! 👩🏿‍💻</p>
+
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:69]: Add a line here which redirects to the CONTRIBUTING.md file. It should be something like `take a look at the <a href = "a
            href="https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md">Contributing Guidelines</a> to contribute to Layer5`

File: _includes/newcomers/newcomers.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    <li>If you need some additional help, please join Layer5 Slack workspace at <a href="http://slack.layer5.io"
+            target="_blank">http://slack.layer5.io</a> and find your way
+        to the <a href="https://layer5io.slack.com/archives/C019426UBNY" target="_blank">#newcomers channel</a>. Feel
+        free to ask questions.</li>
+</ol>
+
+<h4>While your issue is under review</h4>
+
+<ol start="7">
+    <li>Please double-check that you have signed-off on all of your git commits.</li>
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:84]: You have mentioned signing off commits above, I guess we dont need to mention that again here.

File: _includes/newcomers/newcomers.html [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+<p>
+    Whether you are a first-time contributor or an open-source veteran, you are welcome to contribute to and actively
+    engage in the development of projects at Layer5.
+</p>
+<p>
+    Here are some instructions to get you started and if you haven't yet, join the Slack channel to collaborate with the
+    community.
+</p>
+<p>
+    Also, you can check the quick-links below for jumping stright into things.
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:12]: ~stright~ straight

File: _includes/newcomers/newcomers.html [hunk 7]
```diff
@@ -0,0 +1,89 @@
+<h3>Welcome Newcomers!</h3>
+
+<p>
+    Whether you are a first-time contributor or an open-source veteran, you are welcome to contribute to and actively
+    engage in the development of projects at Layer5.
+</p>
+<p>
+    Here are some instructions to get you started and if you haven't yet, join the Slack channel to collaborate with the
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:8]: Add the Slack channel link here please. Btw, awesome work @navendu-pottekkat . You have done some heavy lifting here.

File: _includes/newcomers/newcomers.html [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    <tr>
+        <td><a href="" target="_blank">Calender</a> | <a href="" target="_blank">Slack</a> | <a href=""
+                target="_blank">GitHub</a> | <a href="" target="_blank">YouTube</a> | <a href=""
+                target="_blank">Drive</a></td>
+        <td><a href="" target="_blank">First-timers' Welcome Guide</a></td>
+        <td><a href="" target="_blank">help-wanted</a></td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td><a href="" target="_blank">Welcome Newcomers!</a></td>
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:27]: <a href = "https://docs.google.com/document/d/1tpg2sLxirozNt3Ofr3GdM002f9rExp74EqrsGZBU710/edit#heading=h.rcr9t5pnt3bw"> Welcome Newcomers

File: _includes/newcomers/newcomers.html (hunk 1 above)
Comment (navendu-pottekkat) [reply _includes/newcomers/newcomers.html:28]: Ahh, I forgot to add the links! I was planning to add it at the end. I will fix it

File: _includes/newcomers/newcomers.html [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
+    </tr>
+    <tr>
+        <td><a href="" target="_blank">Weekly Zoom on Thursdays at 8AM Central</a></td>
+        <td><a href="" target="_blank">Layer5 Repository Overview</a></td>
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:32]: https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit#heading=h.504assxl7gpf

File: _includes/newcomers/newcomers.html [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+        <td><a href="" target="_blank">good first issue</a></td>
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:33]: https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22good+first+issue%22

File: _includes/newcomers/newcomers.html (hunk 4 above)
Comment (navendu-pottekkat) [reply _includes/newcomers/newcomers.html:69]: Sure, that seems appropriate here.

File: _includes/newcomers/newcomers.html (hunk 5 above)
Comment (navendu-pottekkat) [reply _includes/newcomers/newcomers.html:84]: Yes. I just used the same steps as in the guide. I will change it.

File: _includes/newcomers/newcomers.html (hunk 7 above)
Comment (navendu-pottekkat) [reply _includes/newcomers/newcomers.html:8]: Sure. I have missed a lot of links. I will fix it. And thank you @tripathyprateek

File: _includes/newcomers/newcomers.html (hunk 10 above)
Comment (navendu-pottekkat) [reply _includes/newcomers/newcomers.html:33]: I forgot to add the links to the table, I will add and commit

**Review by tripathyprateek — CHANGES_REQUESTED** [review]
Awesome work @navendu-pottekkat . I have taken a look, and mentioned some changes. Tagging @chandrashritii to take a look at this once and suggest changes if any.

**Comment by navendu-pottekkat** [comment]
@tripathyprateek The changes you mentioned have been made. I will mark the PR for review.

File: _includes/newcomers/newcomers.html [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+        href="https://layer5.io/community/meshmates">MeshMates</a> and community members are here to help!
+</p>
+
+<h4>Finding an issue to work on</h4>
+
+<ol>
+    <li>Browse the list of <a href="https://github.com/layer5io" target="_blank">Layer5 org</a> issues bearing the “<a
+            href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22good+first+issue%22+"
+            target="_blank">good first issue</a>” label, “<a
+            href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22first+timers+only%22+"
```
Comment (tripathyprateek) [inline _includes/newcomers/newcomers.html:61]: `https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22first-timers-only%22+` is the proper tag to use here

**Review by tripathyprateek — CHANGES_REQUESTED** [review]
Just one last small link change to polish up everything, and the rest looks good to me. Great work @navendu-pottekkat

**Comment by navendu-pottekkat** [comment]
@tripathyprateek I did not notice the broken link. Fixed it

**Review by tripathyprateek — DISMISSED** [review]

File: collections/_pages/newcomers.html [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+                                href="https://youtu.be/wK7Q-zbJ3gQ?t=3">recording</a></td>
+                        <td><a href="/community/members/nupur-thakur">Nupur Thakur</a></td>
+                    </tr>
+                    <tr>
+                        <td>Oct. 15th, 2020</td>
+                        <td><i>A tutorial on contributing to Layer5 and working with Git </i></td>
+                        <td><a
+                                href="https://docs.google.com/presentation/d/1UOlwFtZ-VJhW4RgQoSI_QOGUQ7OBG8NZDWrFUor7eyY/edit?usp=sharing">slides</a>,<a
+                                href="https://youtu.be/O9UZO5g9BvI">recording</a></td>
+                        <td><a href="https://layer5.io/community/members/ruth-ikegah">Ruth Ikegah</a></td>
```
Comment (Jashpatel1) [inline collections/_pages/newcomers.html:55]: Make the links uniform as above.

File: collections/_pages/newcomers.html [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+                    </tr>
+                    <tr>
+                        <td>Oct. 22nd, 2020</td>
+                        <td><i>An introduction to all Layer5 repositories</i></td>
+                        <td><a
+                                href="https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit">slides</a>,<a
+                                href="https://www.youtube.com/watch?v=Yu03slXrdS0&list=PL3A-A6hPO2INVEknQ4R-VwDMUT9yM_FSm&index=8">recording</a>
+                        </td>
+                        <td><a href="https://layer5.io/community/members/lee-calcote">Lee Calcote</a></td>
```
Comment (Jashpatel1) [inline collections/_pages/newcomers.html:64]: Same as above.

File: collections/_pages/newcomers.html [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+                    </tr>
+                    <tr>
+                        <td>Oct. 29th, 2020</td>
+                        <td><i>A tutorial on Gatsby</i></td>
+                        <td><a href="https://www.gatsbyjs.com/docs/quick-start/">slides</a>,<a
+                                href="https://www.youtube.com/watch?v=ug6yaYC-Kkw&list=PL3A-A6hPO2INVEknQ4R-VwDMUT9yM_FSm&index=9">recording</a>
+                        </td>
+                        <td><a href="https://layer5.io/community/members/jash-patel">Jash Patel</a>,<a
```
Comment (Jashpatel1) [inline collections/_pages/newcomers.html:72]: Same as above.

File: collections/_pages/newcomers.html [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+                                href="https://layer5.io/community/members/tanuj-agarwal">Tanuj Agarwal</a></td>
```
Comment (Jashpatel1) [inline collections/_pages/newcomers.html:73]: Same as above.

File: _includes/newcomers/newcomers.html [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+                target="_blank">First-timers' Welcome Guide</a></td>
+        <td><a href="https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+"
+                target="_blank">help-wanted</a></td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td><a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit"
+                target="_blank">Welcome Newcomers!</a></td>
+        <td><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3Ahacktoberfest"
+                target="_blank">Hacktoberfest</a></td>
```
Comment (Jashpatel1) [inline _includes/newcomers/newcomers.html:35]: Hacktoberfest category can be removed. As most of those issues would have been addressed, so no point of adding them under issues for grabs.

**Review by Jashpatel1 — CHANGES_REQUESTED** [review]

File: collections/_pages/newcomers.html (hunk 12 above)
Comment (navendu-pottekkat) [reply collections/_pages/newcomers.html:55]: @Jashpatel1 What do you mean uniform?

Comment (Jashpatel1) [reply collections/_pages/newcomers.html:55]: change the links to /community/members/ruth-ikegah

Comment (navendu-pottekkat) [reply collections/_pages/newcomers.html:55]: You mean point it to the directory and not the web address?

**Comment by navendu-pottekkat** [comment]
@Jashpatel1 @tripathyprateek I have fixed the links and removed the Hacktoberfest link. Please review and merge the changes.

**Review by tripathyprateek — APPROVED** [review]
Thanks @navendu-pottekkat for going through all the hassle once again!

**Comment by Jashpatel1** [comment]
@navendu-pottekkat Great work here. All the things look perfect. Just a minor change in the Tutorials section. For screen widths below 400 px, it is not responsive. Can you look into it and fix it? If you wish to go for another PR for the change, no issues just tell me. Thanks for all the changes till now and also Thanks @tripathyprateek for your valuable feedback.
image

**Comment by tripathyprateek** [comment]
> @navendu-pottekkat Great work here. All the things look perfect. Just a minor change in the Tutorials section. For screen widths below 400 px, it is not responsive. Can you look into it and fix it? If you wish to go for another PR for the change, no issues just tell me. Thanks for all the changes till now and also Thanks @tripathyprateek for your valuable feedback.
> image

@Jashpatel1 we should make a new issue for responsiveness, and get this page merged here😅

**Comment by Jashpatel1** [comment]
> > @navendu-pottekkat Great work here. All the things look perfect. Just a minor change in the Tutorials section. For screen widths below 400 px, it is not responsive. Can you look into it and fix it? If you wish to go for another PR for the change, no issues just tell me. Thanks for all the changes till now and also Thanks @tripathyprateek for your valuable feedback.
> 
> @Jashpatel1 we should make a new issue for responsiveness, and get this page merged here😅

Cool, can you create the issue for it?

**Review by Jashpatel1 — APPROVED** [review]
Thanks for sticking to this for long @navendu-pottekkat . A lot of changes and a new page to the website. We will look forward if you plan to implement this in the layer5-ng website as well.

**Comment by navendu-pottekkat** [comment]
> Thanks for sticking to this for long @navendu-pottekkat . A lot of changes and a new page to the website. We will look forward if you plan to implement this in the layer5-ng website as well.

Yes, we should do that. I would be happy to work on it.

> @Jashpatel1 we should make a new issue for responsiveness, and get this page merged heresweat_smile

Yes we should make it a new issue and fix it.

**Comment by Jashpatel1** [comment]
> > Thanks for sticking to this for long @navendu-pottekkat . A lot of changes and a new page to the website. We will look forward if you plan to implement this in the layer5-ng website as well.
> 
> Yes, we should do that. I would be happy to work on it.
> 
> > @Jashpatel1 we should make a new issue for responsiveness, and get this page merged heresweat_smile
> 
> Yes we should make it a new issue and fix it.

You are free to create the issue and work on it.

**Pull request merged**